dev/com.ibm.websphere.appserver.features/visibility/public/jaxrsClient-3.0/com.ibm.websphere.appserver.jaxrsClient-3.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxrsClient-3.0
+visibility=public
+singleton=true
+IBM-API-Package: javax.ws.rs; type="spec", \
+ javax.ws.rs.container; type="spec", \
+ javax.ws.rs.core; type="spec", \
+ javax.ws.rs.client; type="spec", \
+ javax.ws.rs.ext; type="spec", \
+ javax.ws.rs.sse; type="spec", \
+ com.ibm.websphere.jaxrs20.multipart; type="ibm-api", \
+ com.ibm.websphere.jaxrs.providers.json4j; type="ibm-api"
+IBM-SPI-Package: com.ibm.wsspi.webservices.handler
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-ShortName: jaxrsClient-3.0
+Subsystem-Name: Java RESTful Services Client 3.0
+-features=\
+ com.ibm.websphere.appserver.javax.jaxrs-2.1
+-bundles=\
+  com.ibm.ws.org.jboss.resteasy.resteasy.client, \
+  com.ibm.ws.org.jboss.resteasy.resteasy.client.api, \
+  com.ibm.ws.org.apache.httpcomponents, \
+  com.ibm.ws.org.apache.commons.codec.1.3, \
+  com.ibm.ws.org.apache.commons.logging.1.0.3
+kind=noship
+edition=core
```

Comment:
Should this say `full`?

Reply:
I guess that would be a question for @andymc12, I just copied the jaxrs-2.1 feature and modified it.

Reply:
In which case I am pretty sure it should say `full`. Same applies to the other beta features.

Reply:
updated